README.md

```diff
@@ -1 +1,359 @@
+
+## Create certificates for all these services
+The user facing services are running with TLS encryption. To create the certificates, we used certbot. The certificates are stored in a docker volume, which is mounted in all the containers.
+
+Clone the storage-and-interfaces repo using the following command:
+```bash
+$ git clone https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces.git
+```
+
+Generate a certificate for all external facing services with a command similar to this (be sure to change the email and the domains):
+```bash
+$ /usr/bin/certbot certonly --standalone \
+        --noninteractive \
+        --agree-tos \ 
+        --preferred-challenges http \
+        --email sda-ops@nbis.se \
+        -d htsget.gdi.nbis.se \
+        -d login.gdi.nbis.se \
+        -d download.gdi.nbis.se \
+        -d beacon.gdi.nbis.se \
+        -d inbox.gdi.nbis.se \
+        -d rems.gdi.nbis.se --expand
+```
+
+To renew the certificates and copy them to the docker volume, run the following command or add it to the certbot crontab job, for example:
+```bash
+1 2	1 * * root /usr/bin/certbot renew --deploy-hook /home/ubuntu/starter-kit-storage-and-interfaces/scripts/cert-deploy --quiet --no-self-upgrade
+```
+
+## Deploying storage and interfaces
+Now that the storage and interfaces repo is cloned, start by editing the `config/config.yml` file (a sample can be found under `storage-and-interfaces/config` in this repo) and add the certificates you generated to the “app” entry (which is the download service), like:
+```yaml
+app: # this is for download
+  host: "0.0.0.0"
+  servercert: /shared/cert/fullchain.pem
+  serverkey: /shared/cert/privkey.pem
+  port: "8443"
+```
+
+And to the s3inbox entry in the docker-compose.yml (a sample can be found under `storage-and-interfaces` in this repo) like:
+```yaml
+s3inbox:
+  environment:
+    - SERVER_CERT=/shared/cert/fullchain.pem
+    - SERVER_KEY=/shared/cert/privkey.pem
+```
+
+### Deploying auth
+Auth is a service in the storage-and-interfaces compose file that enables the users to extract a token and an s3config file in order to download and upload data respectively. 
+In the docker-compose.yml, apart from the credentials of the LS AAI account (registered in earlier steps), you should also set the redirect URL, to something like:
+```yaml
+- ELIXIR_REDIRECTURL=https://login.gdi.nbis.se/elixir/login
+```
+
+### Storage
+The Swedish deployment is using an external S3 backend for storing the files. However, the docker compose file in storage-and-interfaces contains a minio instance, that can be used for archiving the data. In either case, set the credentials of the S3 backend in the `config/config.yaml` file, specifically the `S3AccessKey` and `S3SecretKey` values.
+
+### Update the issuer configuration
+Under `config/iss.json` there exists a file that defines the issuer that will be used from the download service. Change this to point to your rems instance, for example:
```

Comment:
```suggestion
Under `config/iss.json` there exists a file that defines the issuer that will be used from the download service. Change this to point to your REMS instance, for example:
```

Also, what is meant by "Under `config/iss.json`"?  Do you possibly mean "In" rather than "Under"?